### PR TITLE
Updating Getting Started Guide for v0.5.0

### PR DESCRIPTION
**Pull request by robscott**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
Now that v0.5.0 is released, we need to update our installation instructions to install the latest version.

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

File: site-src/guides/getting-started.md [hunk 1]
```diff
@@ -25,34 +25,38 @@ network infrastructure that your Gateway resources represent). Note that many
...
+### Install Standard Channel
 
-The following command will install the Gateway API CRDs. This includes
-GatewayClass, Gateway, HTTPRoute, TCPRoute, and more. Note that a running
-Gateway controller in your Kubernetes cluster is required to actually act on
-these resources. Installing the CRDs will just allow you to see and apply the
-resources, but they won't do anything without a controller implementing them.
+The standard release channel includes all resources that have graduated to beta,
+including GatewayClass, Gateway, and HTTPRoute. To install this channel, run the
+following kubectl command:
```
Comment (shaneutt) [inline site-src/guides/getting-started.md:38]:
```suggestion
The standard release channel includes all resources that have at least graduated
to beta, including GatewayClass, Gateway, and HTTPRoute. To install this
channel, run the following kubectl command:
```

**Review by shaneutt — APPROVED** [review]
one small optional comment, otherwise /lgtm

File: site-src/guides/getting-started.md (hunk 1 above)
Comment (robscott) [reply site-src/guides/getting-started.md:38]: I think "at least beta" could confuse users in suggesting that something in our API has graduated beyond beta, maybe we can save this update for when that happens?

Comment (shaneutt) [reply site-src/guides/getting-started.md:38]: ok :+1:

**Pull request merged**